### PR TITLE
Added ESLint

### LINKED ISSUE #2: Install and setup ESLint

### PR DESCRIPTION
**Pull request by nathansit**
Closes #2

### CODE REVIEW

File: eslint.config.mjs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import globals from "globals";
```
Comment (gideontong) [inline eslint.config.mjs]: Can I get the ESLint configuration in something other than `.mjs`, preferably `.js`

**Review by gideontong — CHANGES_REQUESTED** [review]

File: eslint.config.mjs (hunk 1 above)
Comment (nathansit) [reply eslint.config.mjs]: Fine. I'll change the extension and force push. Unless that's bad practice?

Comment (nathansit) [reply eslint.config.mjs]: Nvm will just commit another

**Comment by gideontong** [comment]
File no longer exists

**Comment by gideontong** [comment]
Please also use Typescript compatible ESLint

**Comment by nathansit** [comment]
image

**Comment by gideontong** [comment]
There's no config

**Review by gideontong — APPROVED** [review]

**Pull request merged**